.travis.yml

```diff
@@ -24,6 +24,7 @@ jdk:
   - oraclejdk8
 
 services:
+  - docker
```

Comment:
does caching of docker image downloads happen incidentally, or would we need separate setup for that?